app/components/mig/login/login.js

```diff
@@ -1,5 +1,5 @@
 /*global HTMLElement */
-import closest from './closest'
+import closest from '../../../utils/closest'
```

Comment:
Think this util could be it's own little project. Doesn't have to be done for this pull. But could be used for other projects.